core/src/main/java/com/graphhopper/util/GHUtility.java

```diff
@@ -359,6 +361,18 @@ public static int getEdgeFromEdgeKey(int edgeKey) {
     }
 
     /**
+     * @return a copy of inputArray from [offset, offset+length)
+     */
+    public static LongIndexedContainer copy(LongArrayList inputArray, int offset, int length) {
```

Comment:
Is this faster than construction a LongArrayList with the exptected lenght and this method http://carrotsearch.github.io/hppc/releases/0.7.1/api/com/carrotsearch/hppc/LongArrayList.html#add-long:A-int-int- ?